### PR TITLE
i18n(ko-KR): add `from-gatsby.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- add `from-gatsby.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+Astro의 `<Image />` 컴포넌트는 `.astro` 및 `.mdx` 파일에서만 작동합니다. [컴포넌트 속성의 전체 목록](/ko/guides/images/#속성)을 확인하고 몇 가지 속성이 Gatsby 속성과 다르다는 점에 유의하세요.
+
+표준 Markdown 구문 (`![]()`)을 사용하여 [Markdown (`.md`) 파일의 이미지](/ko/guides/images/#markdown-파일의-이미지)를 계속 사용하려면 링크를 업데이트해야 할 수도 있습니다. HTML `<img>` 태그를 직접 사용하는 것은 로컬 이미지용 `.md` 파일에서 지원되지 않으며 Markdown 구문으로 변환해야 합니다.
+
+```md
+<!-- src/pages/post-1.md -->
+
+# My Markdown Page
+
+<!-- Local image stored at src/assets/stars.png -->
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/migrate-to-astro/from-gatsby.mdx:342]: This can be translated.

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @jsparkdev, I just left a small suggestion 😁

**Comment by jsparkdev** [comment]
thank you It has been modified.

**Review by dreyfus92 — APPROVED** [review]
Thank you @jsparkdev, LGTM ✅

**Pull request merged**